list-targets-with-changed-files/src/run.ts

```diff
@@ -153,16 +153,22 @@ export const run = (input: Input): TargetConfig[] => {
     }
   }
 
+  const moduleCallerMap = JSON.parse(core.getInput("module_callers") || "{}");
```

Comment:
Please don't call `core.getInput` here to make the function easy to test.
Please call `core.getInput` in `main` instead.

Reply:
Ah, that is nice.
Fixed in https://github.com/suzuki-shunsuke/tfaction/pull/1528/commits/e150731c364b7ca740c7729a9054ffad2fa772a2